iphone/Maps/Classes/MapsAppDelegate.mm

```diff
@@ -556,6 +554,9 @@ - (void)applicationWillResignActive:(UIApplication *)application
 {
   [self.mapViewController.appWallAd close];
   [RouteState save];
+  
+  [self.locationManager onBackground];
+  [self.mapViewController onEnterBackground];
```

Comment:
- (void)applicationWillResignActive:(UIApplication *)application
You should use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game. An app in the inactive state should do minimal work while it waits to transition to either the active or background state.